File: front-end/src/constants/OrderStatus.ts
export const OrderStatus = {
    WAITING_PAYMENT: "Chờ thanh toán",
    ALL: "Tất cả",
    PENDING: "Đang chờ",
    CONFIRMED: "Đã xác nhận",
    REJECTED: "Đã hủy",
    CANCELLED: "Đã từ chối",
    DELIVERING: "Đang vận chuyển",
    DELIVERED: "Đã giao",
    COMPLETED: "Đã hoàn thành"
} as const;

export type OrderStatusType = typeof OrderStatus[keyof typeof OrderStatus];

File: back-end/Core/Constants/OrderStatus.cs
namespace back_end.Core.Constants
{
    public static class OrderStatus
    {
        public const string WAITING_PAYMENT = "Chờ thanh toán";
        public const string PENDING = "Đang chờ";
        public const string CONFIRMED = "Đã xác nhận";
        public const string REJECTED = "Đã từ chối";
        public const string CANCELLED = "Đã hủy";
        public const string DELIVERING = "Đang vận chuyển";
        public const string DELIVERED = "Đã giao";
        public const string COMPLETED = "Đã hoàn thành";
    }
}
